docs/en/latest/concepts/annotations.md

```diff
@@ -204,3 +204,54 @@ spec:
             port:
               number: 80
 ```
+
+Use ApisixPluginConfig
+---------
+
+You can use the following annotations to use the `ApisixPluginConfig`.
+
+* `k8s.apisix.apache.org/plugin-conifg-name`
+  
+If this annotations set to `ApisixPluginConfig.metadata.name` the route will enable websoket
```

Comment:
This sentence is weird, why this route will enable websocket?

Reply:
already edited